timeline-chart/src/layer/time-graph-chart-arrows.ts

```diff
@@ -24,13 +24,12 @@ export class TimeGraphChartArrows extends TimeGraphChartLayer {
         if (sourceIndex === -1 || destinationIndex === -1) {
             return undefined;
         }
-        const relativeStartPosition = arrow.range.start - this.unitController.viewRange.start;
         const start: TimeGraphElementPosition = {
-            x: this.getPixel(relativeStartPosition),
+            x: this.getWorldPixel(arrow.range.start),
```

Comment:
same comment as in timeline-chart/src/layer/time-graph-chart-selection-range.ts about APIs.

Reply:
This should also be resolved.